server/controllers/bookController.js

```diff
@@ -1,33 +1,65 @@
 /* eslint-disable require-jsdoc */
+import debug from 'debug';
 import models from '../db/models';
 import Utils from '../helpers/utilities';
 
+const log = debug('dev');
 
 class BookController {
   static async addBook(req, res) {
-    const {
-      title,
-      description,
-      amount,
-      authorID,
-      status,
-      year
-    } = req.body;
+    const { title, description, amount, authorID, status, year } = req.body;
     try {
       const book = await models.Books.create({
         title,
         description,
         amount,
         authorID,
         status,
-        year,
+        year
       });
 
       return Utils.successStatus(res, 201, 'Book added successfully', book);
     } catch (error) {
       Utils.errorStatus(res, 500, error.message);
     }
   }
+
+  static async getBooks(req, res) {
+    if (req.query.page) {
+      let { page, limit } = req.query;
+      page = parseInt(page, 10);
+      page = page > 0 ? page : 1;
+      limit = parseInt(limit, 10);
+      limit = limit > 0 ? limit : 50;
+      const offset = (page - 1) * limit;
+      try {
+        const books = await models.Books.findAndCountAll({
+          limit,
+          offset,
+          $sort: { id: 1 }
+        });
+        const pages = Math.ceil(books.count / limit);
+        const data = books.rows;
+        return res.status(200).json({
+          status: res.status,
+          message: 'success',
+          pages,
+          data
+        });
```

Comment:
Very well done bro. One suggestion, try something out of range like `?page=12&limi=10` you'll see it returns a 'success' message with an empty array.  I think sending the user a different message will be cool.

Reply:
He worked on my task to get all books 😭😭😭😭. Lol. Good work bro. It's just so sad. It really seems like I'm not contributing to this project cuz of electricity. I've worked on this already, I'm just writing tests now tho. Alongside fetch specific book and delete endpoints. It's well